backend/users-service/app/src/index.ts

```diff
@@ -1,7 +1,71 @@
-import { Elysia } from "elysia";
+import { Elysia, t } from 'elysia'
+import { swagger } from '@elysiajs/swagger'
+import { cors } from '@elysiajs/cors'
+import { helmet } from 'elysia-helmet'
+import jwt from '@elysiajs/jwt'
+import { PrismaClient } from '@prisma/client'
+import { logger } from '@bogeychan/elysia-logger'
+import { Role } from './constants'
 
-const app = new Elysia().get("/", () => "Users service").listen(80);
+const db = new PrismaClient()
+
+const app = new Elysia({ prefix: '/users' })
+  .use(cors())
+  .use(helmet())
+  .use(
+    jwt({
```

Comment:
btw what is this `jwt` lib for and what does it do?

Reply:
> btw what is this `jwt` lib for and what does it do?

From my understanding, I would need it later on to verify incoming requests. Yet to implement, but something like:

```
.get('/profile', async ({ jwt, set, cookie: { auth } }) => {
        const profile = await jwt.verify(auth)

        if (!profile) {
            set.status = 401
            return 'Unauthorized'
        }

        return `Hello ${profile.name}`
    })
 ```